### ISSUE TITLE
Console cannot be navigated with a keyboard

### ISSUE THREAD

**Issue by IsabelParedes** [jupyterlab/jupyterlab#18631]
## Description

- The focus is lost after pressing `Esc` in the console input
- There is no logical tab order and the focus gets trapped in the console banner without any indicators
- The content of the console is not scrollable with up/down arrow keys

![Image](https://github.com/user-attachments/assets/287117bb-531f-4e5f-b4e3-845a9eb49a75)

## Reproduce

1. Open the console
2. Press `Esc` to exit the input box
3. Try to navigate with `Tab` or `Shift` + `Tab`

## Expected behavior

- The content of the console should be scrollable with the keyboard
- The focus should not be lost, meaning the focus indicators should be visible
- There should be a logical tab order when using the console

## Context

- Operating System and version: Garuda Linux
- Browser and version: Chromium v145.0.7632.116
- JupyterLab version: v4.5.5

**Comment by Daedalus-Icarus** [comment]
Hi @IsabelParedes I’m interested in contributing to this project and would like to work on this issue as my first contribution. Could you please assign it to me? I’ve reviewed the relevant code and I think I can address this issue

**Comment by IsabelParedes** [comment]
> Hi [@IsabelParedes](https://github.com/IsabelParedes) I’m interested in contributing to this project and would like to work on this issue as my first contribution. Could you please assign it to me? I’ve reviewed the relevant code and I think I can address this issue

Thank you for your enthusiasm, but I already have a solution for this.

**Comment by Gezo18** [comment]
Hi, I’m new to contributing and would like to work on this issue.  
Could you guide me on where to start?
